tools/rebasehelpers/util/git.go

```diff
@@ -343,7 +343,7 @@ func filesInCommit(sha string) ([]File, error) {
 
 func descriptionInCommit(sha string) ([]string, error) {
 	descriptionLines := []string{}
-	stdout, stderr, err := run("git", "show", "--quiet", sha)
+	stdout, stderr, err := run("git", "log", "--pretty=%b", "-1", sha)
```

Comment:
`git log` seems to give you the _real_ field from the commit object:

```
$ git show --pretty=%b
python-libcloud renamed to python2-libcloud, verify fails

$ git log -1 --pretty=%b
python-libcloud renamed to python2-libcloud, verify fails
```

I am more interested in the `--pretty` formatting (for specificity and for not relying on human-readable things like `--quiet`)